pynicotine/gtkgui/userbrowse.py

```diff
@@ -930,9 +930,9 @@ def on_download_directory_to_selected(self, selected, recurse):
     def on_download_directory_to(self, *args, recurse=False):
 
         if recurse:
-            str_title = _("Select Destination for Downloading Folder with Subfolders from User")
+            str_title = _("Select destination folder for this recursive download")
         else:
-            str_title = _("Select Destination for Downloading a Folder from User")
+            str_title = _("Select destination folder for this download")
```

Comment:
Yes! I like this, although it is Missing Title Case.

Reply:
That's not too hard:
`Select destination folder for this download of %(num_files)s, (%(size)s)`

Reply:
Title Case: https://titlecaseconverter.com/

> Title case is a style that is traditionally used for the titles of books, movies, songs, plays, and other works. In title case, all major words are capitalized, while minor words are lowercased. A simple example would be Lord of the Flies.

Reply:
Suggestion for future implementation:_
` Select Destination Folder for Downloading %(num_files)s Files (%(size)s)`